src/Components/VideoCard/VideoCard.jsx

```diff
@@ -10,7 +10,9 @@ export const VideoCard = ({ video }) => {
   return (
     <div key={video._id} className="video-card">
       <div className="for-positioning">
-        <img className="rsp-img" src={video.thumbnail} alt="" />
+        <Link to={`/videos/${video._id}`}>
+          <img className="rsp-img" src={video.thumbnail} alt="" />
+        </Link>
```

Comment:
Suggestion: - Empty alt attribute and can have loading attribute set to lazy

Reply:
adding it right now